src/util/Collection.js

```diff
@@ -349,9 +349,9 @@ class Collection extends Map {
    * @returns {Collection}
    * @example
    * collection
-   *  .tap(coll => coll.size)
+   *  .tap(coll => console.log(`${coll.size} users including bots`))
    *  .filter(user => user.bot)
-   *  .tap(coll => coll.size)
+   *  .tap(coll => console.log(`${coll.size} users without bots`))
```

Comment:
"...users without bots" sounds like we're logging users that don't have a bot.  I recommend changing this to "...users, excluding bots".

Reply:
good suggestion, applied